### PR TITLE
Fix #518 for 03-create

### LINKED ISSUE #518: Missing solutions to, context for, and files required by exercises
Many of the exercises don't have solutions. Some don't need solutions because learners can run the commands and see which is the correct answer. In some of the exercises, the commands can't be tested because the data files don't exist e.g. 07-find, "matching and subtracting" - there are no *ose.dat files

> The -v flag to grep inverts pattern matching, so that only lines which do not match the pattern are printed. Given that, which of the following commands will find all files in /data whose names end in ose.dat (e.g., sucrose.dat or maltose.dat), but do not contain the word temp?
> 
> find /data -name '*.dat' | grep ose | grep -v temp
> find /data -name ose.dat | grep -v temp
> grep -v "temp" $(find /data -name '*ose.dat')
> None of the above.

Sometimes it isn't clear which directory the question refers to.

I plan to submit a series of PRs to address these issues, using existing files as much as possible. Is this of interest to others?

### PR DESCRIPTION
**Pull request by gcapes**
Add solutions, provide working directory context, and where necessary modify the exercises to work with files in data-shell (so that learners can test commands rather than having to work everything out in their heads).

### CODE REVIEW

File: _episodes/03-create.md [hunk 1]
```diff
@@ -556,43 +556,54 @@ but it does find the copy in `thesis` that we didn't delete.
 > {: .output}
 >
 > > ## Solution
```
Comment (shwina) [inline _episodes/03-create.md]: Why get rid of the text explanation?

File: _episodes/03-create.md [hunk 2]
```diff
@@ -556,43 +556,54 @@ but it does find the copy in `thesis` that we didn't delete.
...
 > > {: .bash}
+> > The shell will expand *.dat to match all .dat files in the current directory.
+> > The `mv` command then moves the list of .dat files to the "analyzed" directory.
 > {: .solution}
 {: .challenge}
 
 > ## Copy with Multiple Filenames
 >
-> What does `cp` do when given several filenames and a directory name, as in:
+> For this exercise, you can test the commands in the data-shell/data directory.
```
Comment (shwina) [inline _episodes/03-create.md]: Use backticks: `data-shell/data`

File: _episodes/03-create.md [hunk 3]
```diff
@@ -556,43 +556,54 @@ but it does find the copy in `thesis` that we didn't delete.
...
+>
+> In the example below, what does `cp` do when given several filenames and a directory name?
 >
 > ~~~
+> $ cd ~/data-shell/data
```
Comment (shwina) [inline _episodes/03-create.md]: I would leave out the `cd` here. Many participants will have put the `data-shell` directory in some other location (not `~`).

File: _episodes/03-create.md [hunk 4]
```diff
@@ -556,43 +556,54 @@ but it does find the copy in `thesis` that we didn't delete.
...
 > ~~~
-> $ cp intro.txt methods.txt survey.txt
+> $ cp amino-acids.txt animals.txt morse.txt 
 > ~~~
 > {: .bash}
 >
 > > ## Solution
-> > When given several filenames and a directory name, `cp` copies the files to the destination directory (the last argument).
-> > When given three or more filenames, `cp` expects the last argument to be the destination directory, therefore it returns an error: `target survey.txt is not a directory`.
+> > If given more than one file name followed by a directory name, `cp` copies the files
```
Comment (shwina) [inline _episodes/03-create.md]: Let's make it explicit that the directory name must be the last argument

File: _episodes/03-create.md (hunk 1 above)
Comment (gcapes) [inline _episodes/03-create.md]: I think it should stay after all. Perhaps I was going to add it to the exercise text instead.

**Comment by gcapes** [comment]
Thanks for the review. Do I just rebase onto the same branch, after fixing the points in the comments?

**Comment by gcapes** [comment]
Hi @shwina, 

I hope this is what you intended? I've rebased and fixed the issues addressed in your comments. I had to force push the rebased branch, which means the original commits have disappeared/now show as out of date. 

I also dropped the commit about `Ctrl-R` because that has already been merged and shouldn't have been on this branch. 

Could you confirm this is how I should do the rebasing?
Thanks

**Comment by shwina** [comment]
Yup - it looks good to me. If you are done adding commits, I will merge.

**Comment by gcapes** [comment]
Excellent! All done. I'll let you get back to reviewing the other PRs then ;)

**Pull request merged**